LongWarOfTheChosen/Localization/XComGame.int

```diff
@@ -940,12 +940,15 @@ TacticalText="<Bullet/> Coil weapons automatically pierce (ignore) 1 enemy armor
 AbilityDescName="sniper rifle"
 
 [Shotgun_CG X2WeaponTemplate]
-FriendlyName="Scattergun"
-FriendlyNamePlural="Scatterguns"
+; LWOTC needs translation
```

Comment:
I'm going to be really anal now, but would you mind capitalising "Needs" and "Translation"? It means that people can find the line even with a case-sensitive search.

Reply:
2911afe84d133ba0b30049f310e0d945bc58a5c3